adapters/beachfront/beachfronttest/exemplary/minimal-banner.json

```diff
@@ -57,38 +56,40 @@
           "ua": "",
           "adapterName": "BF_PREBID_S2S",
           "adapterVersion": "0.9.0",
-          "user": {
-          }
+          "user": {}
         }
       },
       "mockResponse": {
         "status": 200,
         "body": [
           {
-            "crid":"crid_1",
-            "price":2.942808,
-            "w":300,
-            "h":250,
-            "slot":"div-gpt-ad-1460505748561-0",
-            "adm":"<div id=\"44861168\"><script>!function(){console.log\"Hello, ad.\";}();<\/script><\/div>"
+            "crid": "crid_1",
+            "price": 2.942808,
+            "w": 300,
+            "h": 250,
+            "slot": "div-gpt-ad-1460505748561-0",
+            "adm": "<div id=\"44861168\"><script>!function(){console.log\"Hello, ad.\";}();<\/script><\/div>"
           }
         ]
       }
     }
   ],
-
-  "expectedBids": [
+  "expectedBidResponses": [
     {
-      "bid": {
-        "id": "12345678",
-        "impid": "test-slot",
-        "price": 2,
-        "adm": "\u003cdiv id=\"12345678\" style=\"width:300;height:250;background-color:black;color:white;\"\u003e300x250\u003c\\div\u003e",
-        "crid": "crid_1",
-        "w": 300,
-        "h": 250
-      },
-      "type": "banner"
+      "bids": [
+        {
+          "bid": {
+            "id": "div-gpt-ad-1460505748561-0Banner",
+            "impid": "div-gpt-ad-1460505748561-0",
+            "price": 2.942808,
+            "adm": "<div id=\"44861168\"><script>!function(){console.log\"Hello, ad.\";}();</script></div>",
```

Comment:
We can't because the these values are set based on the mock response above. I think initially it was a copy paste error but because the tests didn't ever assert the responses, it wasn't caught. 

The, `adm` and `price` values should be exactly what was sent in the mock response from the adapter and the `id` and `impid` values are set based on the `slot` field in their mock response. Ref: https://github.com/prebid/prebid-server/blob/aaecdfada00fa657fcb4237ea2c4caf3b2f664e6/adapters/beachfront/beachfront.go#L581-L583

Reply:
Yes because the json diff tool that we are using for assertion seems to be comparing the unescaped equivalents. I tried modifying the `adm` in the expected response to remove the `</div>` tag and that failed as expected. Also, notice how even though the mock response has escape characters, the `adm` string in the response which is used for comparison doesn't. This is because I believe the string is unescaped before comparison
```
-  "adm": "<div id="44861168"><script>!function(){console.log"Hello, ad.";}();</script></div>",
+  "adm": "<div id="44861168"><script>!function(){console.log"Hello, ad.";}();</script>",
"crid": "crid_1",
```

However, I am gonna update the string to have escape character just for the sake of consistency and to avoid any confusion for the reader. Thanks for pointing it out